package NYT.NTabletServer.NProto;

import "yt/core/misc/guid.proto";

////////////////////////////////////////////////////////////////////////////////

message TReqStartSlots
{
    required NYT.NProto.TGuid cell_id = 1;
    repeated int32 node_ids = 2;
}

message TReqSetCellState
{
    required NYT.NProto.TGuid cell_id = 1;
    required int32 state = 2;
}

message TReqRevokePeer
{
    required NYT.NProto.TGuid cell_id = 1;
    required int32 peer_id = 2;
}

message TReqOnTabletCreated
{
    required NYT.NProto.TGuid tablet_id = 1;
}

message TReqOnTabletRemoved
{
    required NYT.NProto.TGuid tablet_id = 1;
}

////////////////////////////////////////////////////////////////////////////////

